src/OrchardCore.Modules/OrchardCore.Html/Migrations.cs

```diff
@@ -68,7 +68,7 @@ public async Task<int> UpdateFrom3()
             // We are patching all content item versions by moving the Title to DisplayText
             // This step doesn't need to be executed for a brand new site
 
-            var lastDocumentId = 0;
+            long lastDocumentId = 0;
```

Comment:
var lastDocumentId = 0L;

Reply:
I think what I have done is much more readable to me.

Reply:
Maybe, no problem.

Just because Analyzers suggest to replace the `Type` by `var` when it can be inferred by the right part of the assignment, so when it is not the case I'm now used to first try to update the right part.

Reply:
Yes I get it. But in this case I don't think the analyzer complains because the value is set to long on a different line than initializer line. 

If the analyzer shows suggestion, I would have changed it. But I did not recall seeing one

Reply:
> But in this case I don't think the analyzer complains

Indeed if it is not yet inferred from the right part, what I'm now used to do.

I don't want to suggest any personal preference (maybe temporary), only follow the analyzer so no problem when it doesn't complain, but it looks like the trend is to use `var` as much as possible.

Reply:
No worries. You seems to like `var` better. I changed it for you... anything to make @jtkech happy lol

Reply:
Okay cool thanks ;)

On my side I just had a case where it was not possible to use `var`, when the analyzer suggests to remove an useless initialization and just use a simple declaration, we can't use `var someName;`, I thought that the type could have been inferred from the first following assignment.

Reply:
All good. Feel free to approve and/or merge

Reply:
Okay I will look at it this week end.

If you want to take a look at my PRs related to analyzer rules, @hishamco approved one of them but I have 8 PRs related to analyzers.

Reply:
I am actually out on a vacation for couple of weeks. I'll look when I site behind a desk again.